packages/flutter_tools/test/src/fakes.dart

```diff
@@ -398,3 +402,28 @@ class ThrowingPollingDeviceDiscovery extends PollingDeviceDiscovery {
   @override
   bool get canListAnything => true;
 }
+
+class TestPlistParser extends PlistParser {
```

Comment:
Since you re-implement the entire API here, I would `implement` PlistParser` instead, which allows you to remove the dependencies on logger, processManager, and the fileSystem

Reply:
Good call.